File: Examples/VisualBasic/POP3/ConnectingToPOP3.vb
Imports Aspose.Email.Pop3

'
'This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Email for .NET API reference 
'when the project is build. Please check https://Docs.nuget.org/consume/nuget-faq for more information. 
'If you do not wish to use NuGet, you can manually download Aspose.Email for .NET API from http://www.aspose.com/downloads, 
'install it and then add its reference to this project. For any issues, questions or suggestions 
'please feel free to contact us using http://www.aspose.com/community/forums/default.aspx
'

Namespace Aspose.Email.Examples.VisualBasic.Email.POP3
    Class ConnectingToPOP3
        Public Shared Sub Run()
            ' ExStart:GetServerExtensionsUsingPop3Client
            ' Connect and log in to POP3
            Dim client As New Pop3Client("pop.gmail.com", "username", "password")
            client.SecurityOptions = SecurityOptions.Auto
            client.Port = 993
            Dim getCaps As String() = client.GetCapabilities()
            For Each item As String In getCaps
                Console.WriteLine(item)
            Next
            ' ExEnd:GetServerExtensionsUsingPop3Client
        End Sub
    End Class
End Namespace

File: Examples/CSharp/POP3/ConnectingToPOP3.cs
using System;
using Aspose.Email.Clients.Pop3;
using Aspose.Email.Clients;

namespace Aspose.Email.Examples.CSharp.Email.POP3
{
    class ConnectingToPOP3
    {
        public static void Run()
        {
            //ExStart:ConnectingToPOP3
            // Create an instance of the Pop3Client class
            Pop3Client client = new Pop3Client();

            // Specify host, username, password, Port and SecurityOptions for your client
            client.Host = "pop.gmail.com";
            client.Username = "your.username@gmail.com";
            client.Password = "your.password";
            client.Port = 995;
            client.SecurityOptions = SecurityOptions.Auto;
            Console.WriteLine(Environment.NewLine + "Connected to POP3 server.");
            //ExEnd:ConnectingToPOP3
        }
    }
}
